### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by kosssi** [Timshel/vaultwarden#7]
Hello,

Thanks for your PR!

I build a docker image https://hub.docker.com/r/simonc/vaultwarden to test with my OIDC Authelia and It's rocks 👍

I have feedbacks:
- I was expecting with SSO not to need to enter a login but just click on a button.
- I was expecting with SSO not to need to enter a password avec login with my SSO but ultimately I did.
- When I activate `SSO_ONLY: true` I'm not redirect automaticaly on my SSO

Perhaps its features will be added later.

I make my good feedback on vaultwarden chat ;)

**Comment by Timshel** [comment]
Hey,

1.  > I was expecting with SSO not to need to enter a login but just click on a button.

As mentioned in the PR, I made available some custom front-end which improve the flow : cf : https://github.com/Timshel/vaultwarden/blob/main/README.md#docker

2. >  I was expecting with SSO not to need to enter a password avec login with my SSO but ultimately I did.

I suppose you mean a "Master password", cf :

> A master password is still required and not controlled by the SSO (depending of your point of view this might be a feature ;). A key connector to remove this could be added but is not planned in this PR.

3. > When I activate SSO_ONLY: true I'm not redirect automaticaly on my SSO

This could be a nice improvement but since the front-end clients are maintained by Bitwarden, the Vaulwarden team want to keep modification to a minimum.
